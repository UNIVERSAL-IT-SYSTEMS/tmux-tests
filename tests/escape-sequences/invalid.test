#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
printf "\033[abcHa"
printf "\033[abc;defHb"
printf "\033[0;defHc"
printf "\033[abc;0Hd"
printf "\033[0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16;17;18;19;20;21;21;23;24He"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
