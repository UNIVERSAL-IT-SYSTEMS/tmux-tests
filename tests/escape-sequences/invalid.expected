bcHabc;defHbefHcbc;0Hde























